components/constellation/constellation.rs

```diff
@@ -397,8 +396,8 @@ pub struct Constellation<STF, SWF> {
     /// and the namespaces are allocated by the constellation.
     next_pipeline_namespace_id: PipelineNamespaceId,
 
-    /// Bits of state used to interact with the webdriver implementation
-    webdriver: WebDriverData,
+    // Forward responses from the script thread to the webdriver server.
+    input_command_response_sender: Option<IpcSender<WebDriverCommandResponse>>,
```

Comment:
Nit. I think it makes sense to use the name `webdriver_input_command_reponse_sender`, even though it is long, just because someone reading the code should know that this thing is for WebDriver.